### ISSUE TITLE
Crash 1.18.2 - 5.1.0

### ISSUE THREAD

**Issue by EldrinFal** [Caltinor/Project-MMO-2.0#181]
Hi,

Think I ran into a crash caused by PMMO.  Not totally sure, but I'm seeing it referenced in the crash log.  Attaching my log.  Also, not sure if it's related, but lately we've been seeing some mobs moving and attacking really quickly.  Doesn't always happen, but when it does, it's a near guaranteed death.

[crash-2023-01-06_18.07.44-server.txt](https://github.com/Caltinor/Project-MMO-2.0/files/10365328/crash-2023-01-06_18.07.44-server.txt)

**Comment by Caltinor** [comment]
The mobs are being affected by mob scaling. Either dial down the values in pmmo-server.toml or turn them off. 

The crash is caused by using an AOE tool from more vanilla to vein mine. It causes an infinite recursion loop.

**Comment by EldrinFal** [comment]
Which setting do I need to scale down?  I had suspected it was that and
brought the scaling down an extra digit.  Did I move the decimal point in
the wrong direction?  Here is what I have mine at on the two entries that
seemed related:

[Mob_Scaling.Scaling_Settings."Speed Scaling and Ratios"]
combat = 0.0001

[Mob_Scaling.Scaling_Settings."Mob Scaling IDs and
Ratios"."minecraft:generic".movement_speed]
combat = 0.0001

Regarding the more vanilla vein mining, thank you for pointing me in the
right direction.  I have submitted a bug report on their Github for that.
